### PR TITLE
Jupiter 10.14.0 live edge playback

### PR DESCRIPTION
**Pull request by yossipapi**

### CODE REVIEW

File: alpha/apps/kaltura/lib/KSecureEntryHelper.class.php [hunk 1]
```diff
@@ -252,13 +252,38 @@ protected function isFlavorParamsAllowed($flavorParamsId)
 
 	public function updateDeliveryAttributes(DeliveryProfileDynamicAttributes $deliveryAttributes)
 	{
-		$actionList = $this->getActionList(RuleActionType::LIMIT_DELIVERY_PROFILES);
-		if ($actionList)
+		$limitDeliveryProfilesActionList = $this->getActionList(RuleActionType::LIMIT_DELIVERY_PROFILES);
```
Comment (tan-tan-kanarek) [inline alpha/apps/kaltura/lib/KSecureEntryHelper.class.php]: Please move the implementation into the action and run on all actions and call this new implementing method, for example applyDeliveryProfileDynamicAttributes().

File: alpha/apps/kaltura/lib/live/kBroadcastUrlManager.php [hunk 2]
```diff
@@ -99,7 +99,7 @@ protected function getPort ($dc)
 	{
 		$port = kBroadcastUrlManager::DEFAULT_PORT;
 	
-		$broadcastConfig = $this->getConfiguration();	
+		$broadcastConfig = $this->getConfiguration($dc);
```
Comment (tan-tan-kanarek) [inline alpha/apps/kaltura/lib/live/kBroadcastUrlManager.php]: The configuration support port in the top level, once for all DCs, don't enforce here the dc level.

File: alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php [hunk 3]
```diff
@@ -811,6 +811,12 @@ private function serveLiveEntry()
 		}
 		
 		$this->deliveryProfile->setDynamicAttributes($this->deliveryAttributes);	
+
+		if($this->deliveryAttributes->getEdgeServerIds())
```
Comment (tan-tan-kanarek) [inline alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php]: Please move this logic into deliveryProfile->serve.

Comment (yossipapi) [inline alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php]: Will be done later on (while doing refactoring for live delivery profiles)

**Pull request merged**